llvm/test/CodeGen/X86/code_placement_taildup.ll

```diff
@@ -0,0 +1,65 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=x86_64 -enable-ext-tsp-block-placement=1 < %s | FileCheck %s
+
+; Test that dominator tree is invalidated after block removals.
```

Comment:
How is it tested? Does the test trigger assertion without the fix (not able to reproduce it)?

Reply:
The test should trigger the assertion without the fix and without the `MPDT = nullptr` in assignBlockOrder.